examples/README.md

```diff
@@ -0,0 +1,31 @@
+### How to run demo client
+
+1. First start up the narwhal cluster via `docker-compose`
```

Comment:
I would recommend to also point the reader to the full [README](https://github.com/MystenLabs/narwhal/tree/main/Docker) of the docker setup